<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FactorialKrigingDialog</class>
 <widget class="QDialog" name="FactorialKrigingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Factorial Kriging</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>1) Set files for the estimation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frmData">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Data (file, variable):</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmGrid">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Estimation grid:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmVariogram">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Variogram:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnParameters">
        <property name="toolTip">
         <string>Open parameters and run.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/setting16</normaloff>:/icons/setting16</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>2) Review parameters and run:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>5) Save estimates:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3) Map of number of samples:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnShowNSamplesMap">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons32/faciesmap32</normaloff>:/icons32/faciesmap32</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>4) Histogram of number of samples:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnHistogramNSamples">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons32/histo32</normaloff>:/icons32/histo32</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>FactorialKrigingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParameters</sender>
   <signal>clicked()</signal>
   <receiver>FactorialKrigingDialog</receiver>
   <slot>onParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>FactorialKrigingDialog</receiver>
   <slot>onSaveEstimates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShowNSamplesMap</sender>
   <signal>clicked()</signal>
   <receiver>FactorialKrigingDialog</receiver>
   <slot>onMapNSamples()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHistogramNSamples</sender>
   <signal>clicked()</signal>
   <receiver>FactorialKrigingDialog</receiver>
   <slot>onHistoNSamples()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onParameters()</slot>
  <slot>onSaveEstimates()</slot>
  <slot>onMapNSamples()</slot>
  <slot>onHistoNSamples()</slot>
 </slots>
</ui>
